wasm/microwasm/operations.go

```diff
@@ -0,0 +1,591 @@
+package microwasm
```

Comment:
comment somewhere in this file a permalink to the spec for things basically following it. This helps later on updates (ex 1.0 to 1.1 and so forth) as well to understand what is spec-driven vs choice (ex no need to debate naming on things in the spec)